src/compiler/bundle/bundle-output.ts

```diff
@@ -96,6 +96,9 @@ export const getRollupOptions = (
   const afterPlugins = config.rollupPlugins.after || [];
   const rollupOptions: RollupOptions = {
     input: bundleOpts.inputs,
+    output: {
+      inlineDynamicImports: bundleOpts.inlineDynamicImports,
```

Comment:
Minor, can we add JSDoc to `src/compiler/bundle/bundle-interface.ts` where [this field is declared](https://github.com/ionic-team/stencil/blob/b3eedae849c02969a9628f3ea6131d8242222c92/src/compiler/bundle/bundle-interface.ts#L38)?